### PR TITLE
Update and employ the EmptyState component

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Update empty state component](https://github.com/webkom/lego-webapp/commit/bc20c57bd891824e42105e1354440625d407f66f) 

Fixes some bad styling

---

[Employ the EmptyState component various places](https://github.com/webkom/lego-webapp/commit/657afc60f0d663a67bb16b98f192648723e20a01) 

Joblistings and registrations didn't have anything at all.

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Most places look the same without the icon

### CODE REVIEW

**Review by eikhr — APPROVED** [review]
Love it!

File: app/components/EmptyState/index.tsx [hunk 1]
```diff
@@ -4,20 +4,13 @@ import styles from './EmptyState.css';
...
   className?: string;
 };
 
-/**
- * A basic EmptyState component
- *
- */
-const EmptyState = ({ icon, size = 88, className, children }: Props) => (
+const EmptyState = ({ icon, size = 45, className, children }: Props) => (
   <div className={cx(styles.container, icon && styles.centered, className)}>
```
Comment (falbru) [inline app/components/EmptyState/index.tsx:14]: Couldn't a Flex component be used instead?

**Review by falbru — APPROVED** [review]
Looks a lot better!

File: app/components/EmptyState/index.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/components/EmptyState/index.tsx:14]: Meh we only want it "flexed" when there's an icon, so I figured it was best to just keep the `icon && styles.centered`

**Pull request merged**